cmd/wego/flux/cmd.go

```diff
@@ -0,0 +1,34 @@
+package flux
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+//go:embed flux
+var flux []byte
+
+var Cmd = &cobra.Command{
+	Use:   "flux",
+	Short: "Use flux commands",
+	Run:   runCmd,
+}
+
+func init() {
+	_ = os.WriteFile("bin/flux", flux, 0755)
```

Comment:
I think we should use the virtual file system we use in `wks` so we don't have to save anything into the user's file system since we can call it from anywhere.